hammer/service_helpers.py

```diff
@@ -115,7 +115,7 @@ def install_services_cp(services):
     return install_services(prepared_services)
 
 
-def manage_service(names, action):
+def manage_service(names, action, raise_errors=True):
```

Comment:
This will result in automatic documentation (see https://tg-hammer.readthedocs.io/en/latest/api.html#hammer.service_helpers.manage_service)